README.md

```diff
@@ -1,2 +1,39 @@
-# sass-bundle
-Delightful Sass Support for Symfony + AssetMapper
+# Sass For Symfony!
+
+This bundle make it easy to use Sass with Symfony's AssetMapper Component
+(no Node required!).
+
+- Automatically download the correct Sass binary
+- Adds a `sass:build` command to build and watch your sass changes
+
+## Installation
+
+Install the bundle
+
+```shell
+    composer require webmamba/sass-bundle
+```
+
+## Usage
+
+To use generated css, start by including the input file
+(`assets/styles/app.css` by default)
+
+```php
+{# templates.base.html.twig #}
+
+{% block stylesheets %}
+    <link rel="stylesheet" href="{{ asset('styles/app.css') }}">
```

Comment:
I have a proposed change. I also wonder what @bocharsky-bw thinks about this.

Right now, the bundle works by reading `assets/styles/app.scss` and outputting `assets/styles/app.css`. You should then ignore this file from git. We can get fancier. Instead:

A) The user would reference the `app.scss` file directly - `href="{{ asset('styles/app.scss') }}"`
B) Like with TailwindBundle, we would output the Sass process to an internal, temporary file - e.g. `var/sass/app.built.css`
C) Like with TailwindBundle, we would "swap" the contents of the `styles/app.scss` file in AssetMapper for the contents of this `var/sass/app.built.css` file.

The only extra step that this bundle would need that TailwindBundle did not need is:

D) "Remap" the output filename of `styles/app.scss` in AssetMapper. If you do A-C, you will end up with a final file called something like `public/assets/styles/app.12345abcdef.scss` - ending in `.scss`, which we do not want. By decorating the core [PublicAssetsPathResolverInterface](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/AssetMapper/Path/PublicAssetsPathResolverInterface.php), we can change the output name of this file to end in `.css`.

Reply:
Hm, on the one hand, linking to a CSS file is a more straightforward way... but as you said that file should be probably "gitignored", so I think it really better to link directly to the Scss file then excluding working with auto-generated file directly. And I like to make the behavior consistent with TailwindBundle  👍